api/descriptions/policies/vulnerability_serverless_put.md

```diff
@@ -0,0 +1,46 @@
+Updates the vulnerability policy for serverless functions scanned in your pipeline.
```

Comment:
@KICKdesigns Let's use this sentence instead:

Retrieves the vulnerability policy for serverless functions situated in your cloud provider's infrastructure.